azbus/receiver.go

```diff
@@ -139,16 +179,30 @@ func newReceiver(r *Receiver, log Logger, cfg ReceiverConfig, opts ...ReceiverOp
 
 	// Set this to a default that corresponds to the az servicebus default peek-lock timeout
 	if r.Cfg.RenewMessageTime == 0 {
-		r.Cfg.RenewMessageTime = RenewalTime
+		r.Cfg.RenewMessageTime = DefaultRenewalTime
+	}
+
+	if r.Cfg.BatchDeadline == 0 {
+		r.Cfg.BatchDeadline = DefaultRenewalTime
 	}
 
 	return r
 }
 
+// Note: these accessors are intended for batch receivers which take on much more of the responsibility of message lifecycle management.
```

Comment:
yeh, I think this is not great, can't we have hooks here to avoid exposing more and more here?

Reply:
Everything should just be public imo

Reply:
removed